fintech-examples/fintech-impl/pom.xml

```diff
@@ -31,7 +31,11 @@
             <artifactId>mapstruct</artifactId>
             <version>${mapstruct.version}</version>
         </dependency>
-
+        <dependency>
+            <groupId>de.adorsys.opba</groupId>
+            <artifactId>opba-banking-rest-api</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
we must not have any dependency to jars of the opba project. The fintech-server just talks to the interface, so all classes must be available there. you need to generate the interface classes in the pom. So you have to add the tpp_banking_api_token.yml to get the required classes after code generation.

Reply:
I did changes and pushed it. See fintech-impl/pom.xml.

Reply:
thanks.